modules/openapi-generator/src/main/resources/swift5/modelObject.mustache

```diff
@@ -63,7 +63,21 @@
     {{#nonPublicApi}}internal{{/nonPublicApi}}{{^nonPublicApi}}public{{/nonPublicApi}} func encode(to encoder: Encoder) throws {
         var container = encoder.container(keyedBy: CodingKeys.self)
         {{#allVars}}
+        {{#encodeModelNullProperties}}
+        {{#required}}
+        try container.encode{{#isNullable}}IfPresent{{/isNullable}}({{{name}}}, forKey: .{{{name}}})
+        {{/required}}
+        {{^required}}
```

Comment:
@jarrodparkes shouldn't we also take `isNullable` into consideration here?

Reply:
I think I was wrong, and `isNullable` shouldn't be taken into consideration here

Reply:
@4brunu see my comment above — it covers both of your questions